privacyidea/static/components/login/views/login.html

```diff
@@ -56,7 +56,8 @@ <h2 ng-show="piLoginText" class="form-signin-heading">{{ piLoginText }}</h2>
                        class="form-control round-top"
                        ng-model="login.username" required
                        placeholder="{{ 'Username'|translate }}"
-                       focus="true"/>
+                       focus="true"
```

Comment:
Does the `[focus]` here still do anything? If I read correctly, the focus-directive is removed, so this should probably be removed as well.

Reply:
Fixed in 17c361fc